mmcv/version.py

```diff
@@ -32,6 +30,6 @@ def parse_version_info(version_str: str, length: int = 4) -> tuple:
     return tuple(release)
 
 
-version_info = parse_version_info(__version__)
+version_info = tuple(int(x) for x in __version__.split('.')[:3])
```

Comment:
Why do not use parse_version_info here though you wrote it?

Reply:
When you execute this line during setup, "packaging" may not exist.